test/built-ins/String/prototype/codePointAt/position-equal-or-more-than-array-size-returns-undefined.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.3.3
+description: >
+  If pos >= size, return undefined
```

Comment:
I'll leave this out and add a note to make a whole array case on this, as it's good to first prioritize the other string methods.